torch/csrc/jit/runtime/symbolic_script.cpp

```diff
@@ -1440,6 +1440,12 @@ const std::vector<std::string> functions = {
                 return None, None
             return torch.ne(self, other), backward
 
+        def hardshrink(self, lambd: number):
+          def backward(grad_output):
+            mask = ((self > float(lambd)) | (self < -float(lambd))).type_as(self)
```

Comment:
do we not want an `&` here ? https://github.com/pytorch/pytorch/blob/master/aten/src/ATen/native/cpu/Activation.cpp#L421